commercial/app/views/static/adFreeSurveyPage.scala.html

```diff
@@ -44,15 +44,15 @@
 
         <div class="l-side-margins">
 
-            <div class="adfreesurvey-wrapper" data-link-name="adfree survey page">
+            <div class="adfreesurvey-wrapper" data-link-name="live events survey page">
```

Comment:
As this is no longer (as I understand) anything to do with adfree anymore, would we not want to get rid of all these references to adfree in this code? e.g. the 'adfreesurvey-wrapper' and the file name 'adFreeSurveyPage'?